css/tech.scss

```diff
@@ -0,0 +1,229 @@
+.tech{
+	.title {
+		text-align: center;
+		font-size: 2em;
+    margin-bottom: 1%;
+	}
+	.section-1{
+    .hero{
+  		.picture{
+        	position: absolute;
+  			img {
+  				object-fit: cover;
+          width: 100%;
+          height: auto;
+          max-height: 300px;
+  		  }
+  	  }
+      .title{
+        top:138px;
+        color: white;
+      }
+      margin-bottom: 20%;
+    }
+    .section-container{
+      margin-left: 10%;
+      margin-right: 10%;
+      margin-bottom: 3%;
+      position: relative;
+      .section-subhead{
```

Comment:
Please add a blank line before section-subhead here and elsewhere below (whenever you're styling a nested class)